### ISSUE TITLE
order creation - accessibility - City, State and postal code show in one line for some countries

### ISSUE THREAD

**Issue by rossanafmenezes** [woocommerce/woocommerce-android#6079]
As I mentioned [here](https://github.com/woocommerce/woocommerce-android/pull/6040), in the order creation screen - Add Customer details, when you add the address, the city, state/province and zip/postal code are in one line and read together without a proper pause between them and it makes it a bit rushed when you are using talk back. 

After some more testing, I found out this is a bit inconsistent and changes depending on the country you select:

![Annotation on 2022-03-22 at 14-25-48](https://user-images.githubusercontent.com/30724184/159685368-a56858bb-fcea-44a8-ba83-445d8c9203f9.png)

![Screenshot_20220323-072401](https://user-images.githubusercontent.com/30724184/159689984-c20a1caf-6579-4cc2-bae6-fc36d25caf9d.png)

![Screenshot_20220323-072222](https://user-images.githubusercontent.com/30724184/159690007-8bde1697-cda3-4831-ae40-f19d53d2d4fd.png)

![Screenshot_20220323-072337](https://user-images.githubusercontent.com/30724184/159690026-de36c78e-7d89-4dc8-bba4-68932162f461.png)

**Comment by JorgeMucientes** [comment]
Closing this as part of a maintenance pass on the GitHub issue backlog. This issue matched the auto-close criteria for this pass: it has not been updated since 2022-03-23, is unassigned, is not labeled as a bug or high/medium priority, and does not appear to involve POS, payments, login, data loss, crash, security, vulnerability, auth, or fraud.

If this is still relevant, please comment with current details and we can reopen it or move it into Linear for triage.
